src/Database/Schema/SqlserverSchema.php

```diff
@@ -144,19 +164,16 @@ protected function _convertColumn($col, $length = null, $precision = null, $scal
     public function convertColumnDescription(Table $table, $row)
     {
         $field = $this->_convertColumn(
-            $row['type'],
-            $row['char_length'],
-            $row['precision'],
-            $row['scale']
+                $row['type'], $row['char_length'], $row['precision'], $row['scale']
         );
-        if (!empty($row['default'])) {
+        if ( ! empty($row['default'])) {
             $row['default'] = trim($row['default'], '()');
         }
-        if (!empty($row['autoincrement'])) {
+        if ( ! empty($row['autoincrement'])) {
```

Comment:
Also wrong changes. You must really only change what your fix intends to change.